platform/ios/src/MGLScaleBarView.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
+#import <CoreLocation/CoreLocation.h>
+
+@interface MGLScaleBarView : UIView
+
+@property (nonatomic, assign) CLLocationDistance metersPerPoint;
+
+- (void)fadeOut;
```

Comment:
Even though this class is private, it would be good to give it some documentation.